MobileFilter/sections/specific_app.txt

```diff
@@ -5,6 +5,9 @@
 ! Bad:  ||graph.facebook.com^$third-party (should be in adservers.txt)
 !
 !
+! HKTVmall
+||ad-splash-tracking.hktvmall.com^
+||ad-splash.hktvmall.com^
```

Comment:
Wrong file. Please move to `adservers.txt`

Reply:
Fixed